### PR TITLE
[ci] Fixes nightly build for onnx 1.17.1

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: .github/workflows/nightly_publish.yml [hunk 1]
```diff
@@ -123,8 +123,9 @@ jobs:
         run: |
           ./gradlew :integration:test "-Dai.djl.default_engine=PyTorch"
           ./gradlew :integration:clean
-          ./gradlew :integration:test "-Dai.djl.default_engine=OnnxRuntime"
-          ./gradlew :integration:clean
+          # OnnxRuntime 1.17.1 requires GLIBC 2.27
```
Comment (siddvenk) [inline .github/workflows/nightly_publish.yml:126]: Is there another OS we can use when testing with ARM?

GH self-hosted runners will have arm64 machines soon, maybe we just wait for that and we can move away from our self hosted arm64 runners and avoid all these GLIBC version issues

**Review by siddvenk — APPROVED** [review]
